### DISCUSSION TITLE
Multiple values per block

### DISCUSSION THREAD

**Discussion by TheIncredibleChalk** [georgebv/pyextremes#21] [Q&A] [answered]
Hi. 
I wish to extract a fixed number of values (greater than 1) per year. I've built a function that does this. Is there any way I can pass this data to your functions in order to continue the analyses (by fitting a model, etc.)? 

Alternatively, is there a way to set up get_extremes(data, "BM") in order to solve this?

**Comment by georgebv** [comment, marked as answer]
You can use these methods to set extremes manually or create `EVA` object from your custom extremes:

https://github.com/georgebv/pyextremes/blob/f8b097fd49aede2253df23dc654488a540b728d4/src/pyextremes/eva.py#L487

https://github.com/georgebv/pyextremes/blob/f8b097fd49aede2253df23dc654488a540b728d4/src/pyextremes/eva.py#L647

**Reply by TheIncredibleChalk** [reply]
Thank you.
